src/Components/Forms/TestsAndTreatments/Instructions/InstructionsForTreatment.js

```diff
@@ -5,31 +5,122 @@
  * @returns the main form Component.
  */
 
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { FormContext, useForm } from 'react-hook-form';
 import Fields from 'Components/Forms/TestsAndTreatments/Instructions/Fields';
 import PopUpFormTemplates from 'Components/Generic/PopupComponents/PopUpFormTemplates';
 import { connect } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 
+import Grid from '@material-ui/core/Grid';
+import { FHIR } from '../../../../Utils/Services/FHIR';
```

Comment:
change to absulote path

Reply:
import React, { useEffect, useState } from 'react';
import { FormContext, useForm } from 'react-hook-form';
import Fields from 'Components/Forms/TestsAndTreatments/Instructions/Fields';
import PopUpFormTemplates from 'Components/Generic/PopupComponents/PopUpFormTemplates';
import { connect } from 'react-redux';
import { useTranslation } from 'react-i18next';

import Grid from '@material-ui/core/Grid';
import { FHIR } from 'Utils/Services/FHIR';
import denormalizeFhirServiceRequest from 'Utils/Helpers/FhirEntities/denormalizeFhirEntity/denormalizeFhirServiceRequest';

import moment from 'moment';
import SaveForm from 'Components/Forms/GeneralComponents/SaveForm/index';

import { baseRoutePath } from 'Utils/Helpers/baseRoutePath';
import { useHistory } from 'react-router-dom';